src/containers/PipelineRun/PipelineRun.js

```diff
@@ -225,13 +272,52 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
     return (
       <>
-        <RunHeader
-          lastTransitionTime={lastTransitionTime}
-          loading={loading}
-          runName={pipelineRunName}
-          reason={pipelineRunReason}
-          status={pipelineRunStatus}
-        />
+        {/* prettier-ignore */ (showRebuildNotification && !showRebuildNotificationLogsURL) && (
```

Comment:
Prettier complaining about unnecessary parentheses? Let's just keep it happy 😉

Reply:
Prettier removes the brackets which I think changes the functionality as it doesn't evaluate both expressions, lemme check to confirm...

Reply:
My bad, removed the rule, all is well